src/Ormolu/Utils.hs

```diff
@@ -169,3 +173,7 @@ textToStringBuffer txt = unsafePerformIO $ do
   pure StringBuffer {buf, len, cur = 0}
   where
     len = TFFI.lengthWord8 txt
+
+-- | Convert GHC's 'ModuleName' into the one used by Cabal.
+ghcModuleNameToCabal :: GHC.ModuleName -> ModuleName
```

Comment:
Why do we need a wrapper over GHC's module name?

Reply:
We do not have a wrapper over GHC's module name. Instead we use both GHC's `ModuleName` and Cabal's `ModuleName` and therefore have to convert between the two. I do not remember exactly why Cabal's `ModuleName` was preferred for use in fixity maps, but I suspect it might be because it has some useful instances like `NFData` that GHC version doesn't.